docs/howtos/layers/surface.md

```diff
@@ -11,12 +11,15 @@ kernelspec:
   name: python3
 ---
 
-# Using the surface layer
+# Using the `surface` layer
 
-In this document, you will learn about the `napari` `Surface` layer, including
+In this document, you will learn about the `napari` `surface` layer, including
 how to display surface data and edit the properties of surfaces like the
 contrast, opacity, colormaps and blending mode. You will also understand how to
-add and manipulate surfaces both from the GUI and from the console.
+add and manipulate surfaces mostly from the console. There are a few slider
+controls that are available in the GUI.
+
```

Comment:
I think it's worth adding a clear note or warning along the lines of:
`Surface layers can be created only programmatically, i.e. in the console, using a script, etc.`

Reply:
That's a good idea.  It is implied by "After you have added a surface layer programmatically," but why make it obscure?  It's best to be as direct as possible.

Reply:
Did this as a note, too much?